extensions/wikia/Email/Controller/UserNameChangeController.class.php

```diff
@@ -42,19 +42,29 @@ public function body() {
 			'editorAvatarURL' => $this->getCurrentAvatarURL(),
 			'summary' => $this->getMessage( 'emailext-usernamechange-summary' )->text(),
 			'userNameMessage' => $this->getMessage( 'emailext-usernamechange-changed',
-				$this->oldUserName, $this->newUserName )->text(),
+				$this->oldUserName, $this->newUserName )->parse(),
+			'links' => $this->generateLinks(),
 			'contentFooterMessages' => [
 				$this->getMessage( 'emailext-usernamechange-closing' )->text(),
-				$this->getMessage( 'emailext-usernamechange-signature' )->text()
 			],
-			'hasContentFooterMessages' => true
+			'signature' => $this->getMessage( 'emailext-usernamechange-signature' )->text()
 		] );
 	}
 
 	protected function getSubject() {
 		return $this->getMessage( 'emailext-usernamechange-subject' )->text();
 	}
 
+	private function generateLinks() {
+		return [[
+			'label' => $this->getMessage( 'emailext-usernamechange-profile-page' )->text(),
+			'url' => $this->targetUser->getUserPage()->getFullURL()
+		], [
+			'label' => $this->getMessage( 'emailext-usernamechange-check-out' )->text(),
+			'url' => 'http://fandom.wikia.com'
+		]];
+	}
+
 	protected static function getEmailSpecificFormFields() {
 		$form = [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$form". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)